index.html

```diff
@@ -173,126 +173,364 @@ <h2>
+            </dl>
+          </p>
+          <p>
+            In the <a>Pointer Lock State</a> the system mouse cursor must be hidden.
+            Movement and button presses of the mouse must not cause the window
+            to lose focus.
+          </p>
+          <p class="note">
+            Synthetic mouse events created by application script act the same
+            regardless of lock state.
+          </p>
+        </p>
+      </section>
+      <section>
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl data-dfn-for="PointerLockOptions" data-link-for=
+        "PointerLockOptions">
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `"pointerlockchange"` and `"pointerlockerror"` Events
+        </h3>
+        <p class="note">
+          Two events are used to communicate pointer lock state change or an
+          error in changing state. They are named <dfn>pointerlockchange</dfn>
+          and <dfn>pointerlockerror</dfn>.
+        </p>
+        <p class="note">
+          Magnification software increases the size of content on the screen.
+          It uses the mouse to move the magnified point of focus around. When a
+          pointer lock is initiated, the magnification software needs to switch
+          to using the keyboard to move the magnified point of focus around
+          instead. When a pointerlockchange event is fired, web browsers
+          therefore need to make sure the event is communicated to assistive
+          technologies like screen magnifiers.
+        </p>
+      </section>
+      <section>
+        <h3><dfn>Exit Pointer Lock</dfn></h3>
+        <p>
+          The process of exiting pointer lock is as follows:
+          <ol>
+            <li>
+              <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockchange</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a> on the
+                      <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                        User Interaction task source</a>
```

Comment:
You don't fire an event on a task source; you queue a task on a task source. So I think the task source thing needs to move down to the exitPointerLock() method definition.

Finally, there is no "this" in this algorithm, since it is not a Web IDL method definition. You need to pass in a document, or an element, from the calling site.

Reply:
Per our discussion I changed this to Queue a Task and in the method definition I simply referenced this algorithm and passed in the element.

Please take a look and let me know if you think this is in a good state now.